test/extension.test.ts

```diff
@@ -8,38 +8,36 @@ suite('Fabric8 Analytics Extension', () => {
     assert.ok(vscode.extensions.getExtension('redhat.fabric8-analytics'));
   });
 
-  test('should activate', function () {
+  test('should activate', async function () {
     this.timeout(1 * 60 * 1000);
-    return vscode.extensions
+    const api = await vscode.extensions
       .getExtension('redhat.fabric8-analytics')
-      .activate()
-      .then(api => {
-        assert.ok(true);
-      });
+      .activate();
+    assert.ok(true);
   });
 
-  test('should register all fabric8 commands', function () {
-    return vscode.commands.getCommands(true).then(commands => {
-      const FABRIC8_COMMANDS: string[] = [
-        Commands.TRIGGER_FULL_STACK_ANALYSIS,
-        Commands.TRIGGER_FULL_STACK_ANALYSIS_FROM_EDITOR,
-        Commands.TRIGGER_FULL_STACK_ANALYSIS_FROM_EXPLORER,
-        Commands.TRIGGER_FULL_STACK_ANALYSIS_FROM_PIE_BTN,
-        Commands.TRIGGER_FULL_STACK_ANALYSIS_FROM_STATUS_BAR,
-        Commands.TRIGGER_STACK_LOGS
-      ];
-      let foundFabric8Commands = commands.filter(function (value) {
-        return (
-          FABRIC8_COMMANDS.indexOf(value) >= 0 ||
-          value.startsWith('extension.fabric8')
-        );
-      });
-      assert.equal(
-        foundFabric8Commands.length,
-        FABRIC8_COMMANDS.length,
-        'Some fabric8 commands are not registered properly or a new command is not added to the test'
+
+  test('should register all fabric8 commands', async function () {
+    const commands = await vscode.commands.getCommands(true);
+    const FABRIC8_COMMANDS: string[] = [
+      Commands.TRIGGER_FULL_STACK_ANALYSIS,
+      Commands.TRIGGER_FULL_STACK_ANALYSIS_FROM_EDITOR,
+      Commands.TRIGGER_FULL_STACK_ANALYSIS_FROM_EXPLORER,
+      Commands.TRIGGER_FULL_STACK_ANALYSIS_FROM_PIE_BTN,
+      Commands.TRIGGER_FULL_STACK_ANALYSIS_FROM_STATUS_BAR,
+      Commands.TRIGGER_STACK_LOGS
+    ];
+    let foundFabric8Commands = commands.filter(function (value) {
+      return (
+        FABRIC8_COMMANDS.indexOf(value) >= 0 ||
+        value.startsWith('extension.fabric8')
       );
     });
+    assert.equal(
+      foundFabric8Commands.length,
+      FABRIC8_COMMANDS.length,
+      'Some fabric8 commands are not registered properly or a new command is not added to the test'
+    );
```

Comment:
Or use equivalent function from sinon 
```suggestion
      assert.ok(FABRIC8_COMMANDS.includes(...commands);
```

Reply:
ack